CondTools/HLT/test/AlCaRecoTriggerBitsRcdUpdate_cfg_o2o.py

```diff
@@ -0,0 +1,147 @@
+process.MessageLogger.debugModules = cms.untracked.vstring('*')
+# the module writing to DB
+from CondTools.HLT.alCaRecoTriggerBitsRcdUpdate_cfi import alCaRecoTriggerBitsRcdUpdate as _alCaRecoTriggerBitsRcdUpdate
+process.AlCaRecoTriggerBitsRcdUpdate = _alCaRecoTriggerBitsRcdUpdate.clone()
+# The IOV that you want to write out, defaut is 1 to -1/inf. 
+process.AlCaRecoTriggerBitsRcdUpdate.firstRunIOV = options.firstRun # docu see...
+#process.AlCaRecoTriggerBitsRcdUpdate.lastRunIOV = -1 # ...cfi
+# If you want to start from scratch, comment the next line:
+process.AlCaRecoTriggerBitsRcdUpdate.startEmpty = False
+
+# add paths if hlt key has 'special', remove those for the other cases
+runMode = options.hltKey.split('/')[2]
+if(runMode=='special'):
+  print(runMode,'mode: adding',options.pathsToModify,'to',options.keyToModify,'if not exist')
+  process.AlCaRecoTriggerBitsRcdUpdate.pathsToAdd = [
+        cms.PSet(listName = cms.string(options.keyToModify),
+                 hltPaths = cms.vstring(options.pathsToModify.split(','))
+                 )
+  ]
+else: 
+  print('not special mode, removing',options.pathsToModify,'from',options.keyToModify,'if exist') 
+  process.AlCaRecoTriggerBitsRcdUpdate.pathsToRemove = [
+        cms.PSet(listName = cms.string(options.keyToModify),
+                 hltPaths = cms.vstring(options.pathsToModify.split(','))
+                 )
+  ]
+
+process.source = cms.Source("EmptySource",
+                            firstRun = cms.untracked.uint32(options.firstRun) # runnumber
+                            )
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(1) )
+
+process.load("CondCore.CondDB.CondDB_cfi")
+
+# DB input service: 
+process.CondDB.connect = options.inputDB
+process.dbInput = cms.ESSource("PoolDBESSource",
+                               process.CondDB,
+                               toGet = cms.VPSet(cms.PSet(record = cms.string('AlCaRecoTriggerBitsRcd'),
+                                                          tag = cms.string(options.inputTag)
+                                                          )
+                                                 )
+                               )
+
+# DB output service:
+process.CondDB.connect = options.outputDB
+process.PoolDBOutputService = cms.Service("PoolDBOutputService",
+                                          process.CondDB,
+                                          timetype = cms.untracked.string('runnumber'),
+                                          toPut = cms.VPSet(cms.PSet(record = cms.string('AlCaRecoTriggerBitsRcd'),
+                                                                     tag = cms.string(options.outputTag) # choose output tag you want
+                                                                     )
+                                                            )
+                                          )
+
+# Put module in path:
+process.p = cms.Path(process.AlCaRecoTriggerBitsRcdUpdate)
+
```

Comment:
@hjkwon260 if this is the script used in the O2O deployment, it should better go in a /python folder rather than /test

Reply:
/python has been removed from the package. shall I create one and move the script?

Reply:
@hjkwon260 please clarify if the suggested changes could be implemented in time for inclusion of this PR by the 12_3_0_pre6 deadline ( [2022/03/08](https://twiki.cern.ch/twiki/bin/view/CMS/CMSSW_12_3_0) ). Thank you.